doc-content/enterprise-only/installation/patches-applying-proc.adoc

```diff
@@ -18,10 +23,10 @@ NOTE: Only updates for {PRODUCT} are included in {PRODUCT} update tools. Updates
 Example:
 
 * *Product:* {PRODUCT_SHORT}
-* *Version:* {PRODUCT_VERSION}.1
+* *Version:* {PRODUCT_VERSION}.0
 --
 +
-If you are upgrading to a new minor release of {PRODUCT}, such as an upgrade from version {PRODUCT_VERSION}.x to 7.4, first apply the latest patch update to your current version of {PRODUCT} and then follow this procedure again to upgrade to the new minor release.
+If you are upgrading to {PRODUCT} 7.4.x from 7.2.0 or higher, first apply the 7.3.1 patch update to your current version of {PRODUCT} and then follow this procedure again to upgrade to 7.4.x.
```

Comment:
This does not sound right. If one has 7.2.0 then 7.3.1 is not a "patch update"?

Reply:
Yeah, it is right.

Reply:
@mcimbalek please confirm that the sentence that Misha is questioning is correct.